pkgs/applications/misc/msglint/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "msglint";
+  version = "1.04";
+
+  sourceRoot = ".";
+
+  src = fetchurl {
+    url = "https://tools.ietf.org/tools/msglint/msglint-${version}.tar.gz";
+    sha256 = "1yasj6nijip3z86m526bsssd5vgqrx233azvpz1c0z564wngvzbm";
+  };
+
+  patches = [ ./fixfree.patch ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp msglint $out/bin/msglint
```

I think this should work:

```suggestion
    runHook preInstall
    
    install msglint $out/bin/msglint
    
    runHook postInstall
```

---

should or can ? :)